grading/load_config_json.cpp

```diff
@@ -1297,6 +1315,7 @@ void AddSubmissionLimitTestCase(nlohmann::json &config_json) {
     limit_test["type"] = "FileCheck";
     limit_test["title"] = "Submission Limit";
     limit_test["max_submissions"] = MAX_NUM_SUBMISSIONS;
+    limit_test["testcase_id"] = "submission_limit_test";
```

Comment:
Are we making sure somewhere that no one manually named their testcase "submission_limit_test"
(or if they did, it was actually overriding the submisison test and not some other test)